### PR TITLE
Fix hardcoded nodeAttestor and keyManager in spire-agent

### PR DESCRIPTION
**Pull request by kfox1111**
Fixes: https://github.com/spiffe/helm-charts-hardened/issues/220

### CODE REVIEW

**Review by marcofranssen — APPROVED** [review]
LGTM

File: charts/spire/charts/spire-agent/templates/configmap.yaml [hunk 1]
```diff
@@ -56,15 +56,29 @@ agent:
   {{- end }}
   {{- end }}
 
+{{- $nodeAttestorUsed := add (len .Values.customPlugins.nodeAttestor) (len .Values.unsupportedBuiltInPlugins.nodeAttestor) }}
+{{- $keyManagerUsed := add (len .Values.customPlugins.keyManager) (len .Values.unsupportedBuiltInPlugins.keyManager) }}
```
Comment (faisal-memon) [inline charts/spire/charts/spire-agent/templates/configmap.yaml:60]: How are we handling this conflict elsewhere?

Comment (kfox1111) [reply charts/spire/charts/spire-agent/templates/configmap.yaml:60]: This code was basically copy/pasted from the spire-server where we are already doing it.

Comment (kfox1111) [reply charts/spire/charts/spire-agent/templates/configmap.yaml:60]: https://github.com/spiffe/helm-charts-hardened/blob/main/charts/spire/charts/spire-server/templates/configmap.yaml#L109-L150

Comment (faisal-memon) [reply charts/spire/charts/spire-agent/templates/configmap.yaml:60]: Thanks for clarifying.

**Review by faisal-memon — APPROVED** [review]

**Pull request merged**